### ISSUE TITLE
nginx not reachable

### ISSUE THREAD

**Issue by oneiros66** [mailcow/mailcow-dockerized#680]
Hi,

since short, I try to host mailcow dockerized on a netcup root server.
Therefore i run Debian 8 with no SELinux enabled.
Docker Version is 17.10.0 Docker Compose Version is 1.16.1, build 6d1ac21

I started the server as normal by pulling it, creating a basic configuration, changing the listening ports from 80 to 8088 and from 443 to 4438 - I plan on using a reverse proxy later on.

The main problem is: I can't reach the nginx-webinterface (website is not reachable), neither per http or https.
The port isn't blocked and I'm sure its opened (checked it with netstat). 
nginx is also listening on the right port (as defined).

In the docker-compose log of nginx I found some things that confuse me:
rspamd-mailcow_1     | 2017-10-23 21:12:19 #17(controller) <9fydgy>; map; http_map_error: error reading http://172.22.1.251:8081/settings.php(172.22.1.251:8081): connection with http server terminated incorrectly: IO timeout

The only thing I'm getting out of nginx-logs is that nginx is attaching to all of the containers. No errors or anything else suspicious. 

I will of cause post more information if needed and I'm more than only willing to donate if it's workin again ;) 

Thanks!

**Comment by jas8522** [comment]
netstat will show you when a port is bound, but it can't tell you if it's externally available. Check iptables? ```iptables -L -n```

If there's anything in there, for testing at least, run iptables -F to clear it out.

**Comment by oneiros66** [comment]
Thanks for your reply! 
I checked iptables, they look fine. I flushed 'em, of cause i destroyed my other docker-rules 2 (have an onlyoffice running integrated, restarted the docker-service 2 make sure, the rules are rewritten and restarted the mailcow-dockerized packages via docker-compose. 
IP-tables look nearly the same as before and in detail they look like the following: 

[iptables_actual.txt](https://github.com/mailcow/mailcow-dockerized/files/1423814/iptables_actual.txt)

They look fine to me and nothing changed, still can't reach the webhost...
